tensorflow_recommenders_addons/dynamic_embedding/core/kernels/lookup_impl/lookup_table_op_cpu.h

```diff
@@ -138,12 +139,12 @@ class TableWrapperBase {
   virtual Status export_values(OpKernelContext* ctx, int64 value_dim) {
     return Status::OK();
   }
-  virtual Status save_to_hdfs(OpKernelContext* ctx, int64 value_dim,
-                              const string& filepath,
-                              const size_t buffer_size) {}
-  virtual Status load_from_hdfs(OpKernelContext* ctx, int64 value_dim,
-                                const string& filepath,
-                                const size_t buffer_size) {}
+  virtual Status save_to_file_system(OpKernelContext* ctx, int64 value_dim,
```

Comment:
It looks like only for CPU? how about GPU users?

Reply:
GPU version is done now.